src/main/scala/com/ubirch/webui/models/keycloak/util/BareKeycloakUtil.scala

```diff
@@ -569,6 +598,35 @@ case class MemberResourceRepresentation(resource: UserResource, representation:
 
   }
 
+  /**
+    * Calling this road will:
+    * - Check that the device is claimed and owned by the user making the request
+    * - Remove it from groups (OWN_DEVICES_user, FIRST_CLAIMED_user, CLAIMED_provider)
+    * - Add it to the group UNCLAIMED_DEVICES
+    * - Remove relevant device attributes: FIRST_CLAIMED_TIMESTAMP, claiming_tags
+    */
+  def unclaimDevice(): MemberResourceRepresentation = {
+
+    val unclaimedDevicesGroup = GroupFactory.getByName(Elements.UNCLAIMED_DEVICES_GROUP_NAME).toGroupFE
```

Comment:
What is GroupFE especially FE? Maybe I shoudl check out the project next time. It's easier to review with more context..